test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlWriterTests.cs

```diff
@@ -2460,6 +2460,77 @@ private void WriteAndVerifyXml(IEdmModel model, string expected, CsdlTarget targ
             }
         }
 
+        [Fact]
+        public void VerifyKeyAliasForComplexPropertiesIsSupported()
+        {
+            // Arrange
+            var model = new EdmModel();
+            var entity = new EdmEntityType("NS1", "Product");
+            var complexType = new EdmComplexType("NS1", "ExampleComplexObject");
+            complexType.AddStructuralProperty("Name", EdmCoreModel.Instance.GetString(false));
+            var entityId = entity.AddStructuralProperty("ExampleObject", new EdmComplexTypeReference(complexType, isNullable: false));
+            entity.AddKeys(entityId);
+            entity.AddKeyAlias(new KeyValuePair<IEdmStructuralProperty, string>(entityId, "ExampleObjectName"));
+            entity.AddStructuralProperty("Name", EdmCoreModel.Instance.GetString(false));
+            entity.AddStructuralProperty("UpdatedTime", EdmCoreModel.Instance.GetDate(false));
+            model.AddElement(entity);
+
+            var entityContainer = new EdmEntityContainer("NS1", "Container");
+            model.AddElement(entityContainer);
+            EdmEntitySet set1 = new EdmEntitySet(entityContainer, "Products", entity);
+            entityContainer.AddElement(set1);
+
+            // Act & Assert for XML
+            WriteAndVerifyXml(model, "<?xml version=\"1.0\" encoding=\"utf-16\"?>" +
+                "<edmx:Edmx Version=\"4.0\" xmlns:edmx=\"http://docs.oasis-open.org/odata/ns/edmx\">" +
+                  "<edmx:DataServices>" +
+                    "<Schema Namespace=\"NS1\" xmlns=\"http://docs.oasis-open.org/odata/ns/edm\">" +
+                      "<EntityType Name=\"Product\">" +
+                        "<Key>" +
+                          "<PropertyRef Name=\"NS1.ExampleComplexObject/ExampleObject\" Alias=\"ExampleObjectName\" />" +

```

Comment:
I think the name of the property ref in this case should be `<NameOfComplexProperty>/<NameOfReferencedProperty>` not ~`<TypeOfProperty>/<NameOfProperty>`~. So a valid example would have been:
```xml
<PropertyRef Name="ExampleObject/Name" Alias="ExampleObjectName" />
```

Reply:
yes. seen this is an error